docs/components/concepts/clusters.md

```diff
@@ -70,19 +70,15 @@ When your Free Trial plan expires, you are automatically transferred to the Free
 
 ### Auto-pause
```

Comment:
Maybe outside of the scope of this issue, but could be worth noting or linking to on https://docs.camunda.io/docs/next/components/console/manage-clusters/manage-cluster/.

Reply:
Thanks, I'll note that as maybe part of a new PR 👍